tests/tracer/test_logging.py

```diff
@@ -0,0 +1,284 @@
+import os
+
+
+def test_unrelated_logger_in_debug_with_patch_all(run_python_code_in_subprocess, tmpdir):
+    """
+    When importing the tracer after logging has been configured,
+                with debug mode and enabling a custom log path,
+                the ddtrace logger does not override any custom logs settings and
+                the ddtrace logger continues to have its expected configuration and
+                logs to a custom file path.
+    """
+    env = os.environ.copy()
+    env["DD_TRACE_DEBUG"] = "true"
+    log_file = tmpdir + "/testlog.log"
+    env["DD_TRACE_LOG_FILE"] = log_file
```

Comment:
we should be able to test max bytes here as well (or in another test, we do want to test the default values)

```python
env["DD_TRACE_FILE_SIZE_BYTES"] = "20000"  # random example
```

then can do the same assertion you have below, but with `20000`

Reply:
Marking this as resolved since there are now tests for `DD_TRACE_FILE_SIZE_BYTES` too.